logbook-spring-webflux/pom.xml

```diff
@@ -40,6 +40,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty.ee10</groupId>
+            <artifactId>jetty-ee10-servlet</artifactId>
```

Comment:
Is it required for tests? I'd expect that wiremock-standalone would have it already

Reply:
Same as above; this is now no longer needed.